packages/@sanity/imagetool/src/HotspotImage.js

```diff
@@ -21,9 +21,8 @@ export default class HotspotImage extends React.PureComponent {
     src: PropTypes.string.isRequired,
     srcAspectRatio: PropTypes.number.isRequired,
     srcSet: PropTypes.string,
-    hotspot: PropTypes.object.isRequired,
-    crop: PropTypes.object.isRequired,
-    width: PropTypes.number,
+    hotspot: PropTypes.PropTypes.objectOf(PropTypes.object),
+    crop: PropTypes.PropTypes.objectOf(PropTypes.object),
```

Comment:
Part of the linting changes, I was getting errors which lead me to believe that the `.isRequired` was the type which lead to more errors on the definition of the `defaultProps`